scripts/badger/process_bribes_bvecvx.py

```diff
@@ -41,10 +41,15 @@ def step0_1(sim=False):
         claimed = {alcx.address: 500e18}
     else:
         claimed = SAFE.badger.claim_bribes_votium(registry.eth.bribe_tokens_claimable)
-        # Handling cvxFXS rewards as bribes (Ref: https://github.com/Badger-Finance/badger-strategies/issues/56)
-        cvxFXS_amount = SAFE.badger.sweep_reward_token(CVX_FXS)
-        if cvxFXS_amount > 0:
-            claimed[CVX_FXS] = claimed.setdefault(CVX_FXS, 0) + cvxFXS_amount
```

Comment:
Pretty smart way to generalize the sweeping! I can confirm that the badger wrapper function will check if the balance for each token is 0 and, if this is the case, it will simply return 0 so the new logic should handle it properly.

Any new reward tokens emitted to vlCVX will simply need to be added to the bribe_tokens_claimable list. Downside of this is that the script will take a bit longer to run to potentially only handle one known token as everything else will be transferred on claim_bribes_votium. Not a bad tradeoff in favor of generalization!

Reply:
LG!